docs/pages/repo/docs/core-concepts/monorepos/running-tasks.mdx

```diff
@@ -270,6 +270,34 @@ be explicitly opted into via including `//#<task>` in the pipeline configuration
 some best-effort checking to produce an error in the recursion situations, but it is up to you to only
 opt in those tasks which don't themselves trigger a `turbo` run that would recurse.
 
+### Dependencies outside of a task
+  
+When your task has topological dependencies that are outside of that given task, you'll still want to enjoy the parallelism of Turborepo and ensure that your caching behavior is correct according to your code changes.
+  
+To demonstrate how to do this, let's say you have a set of workspaces to do a little bit of math: `add`, `subtract`, and `multiply`. You have tests in all three workspaces and you're using the `add` package in `subtract` and `multiply`. We have these packages set up as internal dependencies so they can be used directly in other workspaces.
```

Comment:
```suggestion
To demonstrate how to do this, let's say you have a set of workspaces to do a little bit of math: `add`, `subtract`, and `multiply`. Because you're a clever engineer, you implemented `subtract` by calling `add` with a negative number, and `multiply` by calling `add` in a loop. You have tests in all three workspaces and—since you're using the `add` package in `subtract` and `multiply`—any changes to `add` should cause `subtract` and `multiply` to rerun. In other words, `add` is set up as an internal dependency directly used in both of the `subtract` and `multiply` workspaces.
```